src/common/Widget.js

```diff
@@ -3,6 +3,20 @@
     if (typeof define === "function" && define.amd) {
         define(["d3/d3"], factory);
     } else {
+        root.require = root.require || function (paths, cb) {
```

Comment:
I've just committed a full non-AMD demo and submitted a pull request to your repo. Funny thing I've found out is that we actually don't have to tweak the require shim at all, it would just work because the return value of d3-cloud module isn't used anywhere in the code.